src/TagBoxContainer.jsx

```diff
@@ -15,14 +15,16 @@ export default class TagBoxContainer extends Component {
     renderTag: PropTypes.func,
     loadingText: PropTypes.string,
     selectedText: PropTypes.string,
-    placeholder: PropTypes.string
+    placeholder: PropTypes.string,
+    search: PropTypes.func
   }
 
   static defaultProps = {
     renderNewOption: input => `Add ${input}...`,
     loadingText: 'Loading...',
     selectedText: 'Already Selected',
-    placeHolder: ''
+    placeHolder: '',
+    search: (tag, input, selected) => tag.label.includes(input) && !selected.includes(tag)
```

Comment:
Ok. I need to sleep on this. What this means is that if you override the default `search` function, you are going to see tags that you've already selected unless you also do a check for `selected.includes(tag)`. I would _think_ that you would *never* want to see tags you'e already selected, regardless of what you're using for a `search` function. 

On the other hand, maybe that's the UX that someone wants to provide. If so, there might be a better way of allowing it, which I might consider for a future version.